src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -60,17 +60,13 @@ using ::testing::ReturnRefOfCopy;
 
 // For service types and PROTECTION_ON/OFF
 
-enum UnnamedService {
-    served_service1 = 0x06,
-   served_service2 = 0x08
-};
+enum UnnamedService { served_service1 = 0x06, served_service2 = 0x08 };
```

Comment:
@AByzhynar google style: Enumerators should be named either like constants or like macros: either kEnumName or ENUM_NAME.  Also notice that beautifier doesn't preserve braces align.